src/FileFormats.Minidump/MinidumpStructures.cs

```diff
@@ -0,0 +1,163 @@
+        public uint DataSize;
+        public uint Rva;
+    }
+
+    internal enum MINIDUMP_STREAM_TYPE
+    {
+        UnusedStream = 0,
+        ReservedStream0 = 1,
+        ReservedStream1 = 2,
+        ThreadListStream = 3,
+        ModuleListStream = 4,
+        MemoryListStream = 5,
+        ExceptionStream = 6,
+        SystemInfoStream = 7,
+        ThreadExListStream = 8,
+        Memory64ListStream = 9,
+        CommentStreamA = 10,
+        CommentStreamW = 11,
+        HandleDataStream = 12,
+        FunctionTableStream = 13,
+        UnloadedModuleListStream = 14,
+        MiscInfoStream = 15,
+        MemoryInfoListStream = 16,
+        ThreadInfoListStream = 17,
+        LastReservedStream = 0xffff,
+    }
+
+
+    internal class MINIDUMP_SYSTEM_INFO : TStruct
+    {
+        public ProcessorArchitecture ProcessorArchitecture;
+        public ushort ProcessorLevel;
+        public ushort ProcessorRevision;
+        public byte NumberOfProcessors;
+        public byte ProductType;
+        public uint MajorVersion;
+        public uint MinorVersion;
+        public uint BuildNumber;
+        public uint PlatformId;
+        public uint CSDVersionRva;
+    }
+
+    internal enum ProcessorArchitecture : ushort
+    {
+        PROCESSOR_ARCHITECTURE_INTEL = 0,
+        PROCESSOR_ARCHITECTURE_MIPS = 1,
+        PROCESSOR_ARCHITECTURE_ALPHA = 2,
+        PROCESSOR_ARCHITECTURE_PPC = 3,
+        PROCESSOR_ARCHITECTURE_SHX = 4,
+        PROCESSOR_ARCHITECTURE_ARM = 5,
+        PROCESSOR_ARCHITECTURE_IA64 = 6,
+        PROCESSOR_ARCHITECTURE_ALPHA64 = 7,
+        PROCESSOR_ARCHITECTURE_MSIL = 8,
+        PROCESSOR_ARCHITECTURE_AMD64 = 9,
+        PROCESSOR_ARCHITECTURE_IA32_ON_WIN64 = 10,
+    }
+
+    internal sealed class VS_FIXEDFILEINFO : TStruct
+    {
+        public uint dwSignature;            /* e.g. 0xfeef04bd */
+        public uint dwStrucVersion;         /* e.g. 0x00000042 = "0.42" */
+        public uint dwFileVersionMS;        /* e.g. 0x00030075 = "3.75" */
+        public uint dwFileVersionLS;        /* e.g. 0x00000031 = "0.31" */
+        public uint dwProductVersionMS;     /* e.g. 0x00030010 = "3.10" */
+        public uint dwProductVersionLS;     /* e.g. 0x00000031 = "0.31" */
+        public uint dwFileFlagsMask;        /* = 0x3F for version "0.42" */
+        public uint dwFileFlags;            /* e.g. VFF_DEBUG | VFF_PRERELEASE */
+        public uint dwFileOS;               /* e.g. VOS_DOS_WINDOWS16 */
+        public uint dwFileType;             /* e.g. VFT_DRIVER */
+        public uint dwFileSubtype;          /* e.g. VFT2_DRV_KEYBOARD */
+
+        // Timestamps would be useful, but they're generally missing (0).
+        public uint dwFileDateMS;           /* e.g. 0 */
+        public uint dwFileDateLS;           /* e.g. 0 */
+    }
+
+
+    internal sealed class MINIDUMP_LOCATION_DESCRIPTOR : TStruct
+    {
+        public uint DataSize;
+        public uint Rva;
+    }
+
+    [TStructPack(4)]
```

Comment:
One issue you are going to run into here is you've defined TStruct and friends as public.  "Pack" is such a common term that I worry you'll run into collisions with "PackAttribute", hence why I named it something off-key.  However, I suppose that's why namespaces exist...I've changed it.